<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>349</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>201</width>
      <height>121</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QPushButton" name="btnDepthClosedLoop">
       <property name="text">
        <string>深度闭环</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="btnSet0Depth">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>设为0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QPushButton" name="btnRefreshDepth">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>更新深度</string>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QPushButton" name="btnCourseClosedLoop">
       <property name="text">
        <string>航向闭环</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="courseVal">
       <property name="wrapping">
        <bool>true</bool>
       </property>
       <property name="minimum">
        <double>-180.000000000000000</double>
       </property>
       <property name="maximum">
        <double>180.000000000000000</double>
       </property>
       <property name="singleStep">
        <double>5.000000000000000</double>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>°</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="btnUseCurYaw">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>设为当前角度</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QPushButton" name="btnRefreshCourse">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>更新航向</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>cm</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="2">
      <widget class="QDoubleSpinBox" name="depthVal">
       <property name="decimals">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>475</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>日志</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuComm">
    <property name="title">
     <string>通信</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuChart">
    <property name="title">
     <string>图表</string>
    </property>
    <widget class="QMenu" name="menu">
     <property name="title">
      <string>显示点数</string>
     </property>
     <addaction name="act10Points"/>
     <addaction name="act20Points"/>
     <addaction name="act50Points"/>
     <addaction name="act100Points"/>
    </widget>
    <addaction name="menu"/>
    <addaction name="actChartAniSwitch"/>
   </widget>
   <addaction name="menuRecord"/>
   <addaction name="menuComm"/>
   <addaction name="menuChart"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actChartAniSwitch">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>动画(尚不完善）</string>
   </property>
   <property name="toolTip">
    <string>图表动画开关</string>
   </property>
  </action>
  <action name="act10Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>10 (1point / s)</string>
   </property>
  </action>
  <action name="act20Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>20 (2points / s)</string>
   </property>
  </action>
  <action name="act50Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50 (5points / s)</string>
   </property>
  </action>
  <action name="act100Points">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100 (10points / s)</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
